src/components/menu/menu-segment-title/menu-segment-title.spec.tsx

```diff
@@ -39,6 +41,17 @@ describe("Title", () => {
     expect(wrapper.find(StyledTitle).prop("menuType")).toBe("light");
   });
 
+  it("should title as an h2 element by default and render list items as children of the ul", () => {
+    wrapper = render("light");
+    openSubmenu(wrapper);
+    const title = wrapper.find(StyledTitle);
+    const segmentList = wrapper.find(StyledSegmentChildren);
+
+    expect(title.getDOMNode().tagName).toBe("H2");
+    expect(segmentList.getDOMNode().tagName).toBe("UL");
+    expect(segmentList.find("li").text()).toBe("bar");
```

Comment:
**thought**: would it be worth us as a team adopting an approach of querying DOM elements first over React DOM nodes where possible?

This would make migrating to RTL easier given it forces you to write tests in a way that tests the final markup a user would interact with.

```suggestion
    expect(title.getDOMNode().tagName).toBe("H2");
    expect(segmentList.getDOMNode().tagName).toBe("UL");
    expect(segmentList.find("li").text()).toBe("bar");
```

Reply:
yeah my thinking is this can easily be converted when we move to RTL